sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ReceiveMessageOptions.java

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
`ReceiveMessageOptions` :  Should this be removed ? You have added these properties in  ServiceBusReceiverBuilder already.

Reply:
This is a package-private class. It's easier than passing around 6 parameters.

Reply:
Okay, Its fine. This package-private. No need to take it out.